index.html

```diff
@@ -1,203 +1,576 @@
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            class="lucide lucide-zap"
+          >
+            <path
+              d="M4 14a1 1 0 0 1-.78-1.63l9.9-10.2a.5.5 0 0 1 .86.46l-1.92 6.02A1 1 0 0 0 13 10h7a1 1 0 0 1 .78 1.63l-9.9 10.2a.5.5 0 0 1-.86-.46l1.92-6.02A1 1 0 0 0 11 14z"
+            ></path>
+          </svg>
+          <p>Current version | v10.2.0</p>
+        </div>
+        <h1
+          class="mt-6 text-6xl font-semibold tracking-tighter text-gray-800 sm:text-7xl"
+        >
+          Gridstack.js
+        </h1>
+
+        <p class="mt-8">Build interactive dashboards in minutes</p>
+
+        <!-- Github button link -->
+        <a
+          href="https://github.com/gridstack/gridstack.js"
+          target="_blank"
+          class="inline-flex items-center h-16 gap-2 px-8 mt-8 text-lg font-semibold text-white transition-all rounded-full shadow-inner bg-gradient-to-br from-indigo-400 to-indigo-600 shadow-white/25 group hover:scale-95 hover:opacity-80"
+        >
+          <!-- Github Icon -->
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            stroke="currentColor"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            class="transition-all lucide lucide-github group-hover:scale-110"
+          >
+            <path
+              d="M15 22v-4a4.8 4.8 0 0 0-1-3.5c3 0 6-2 6-5.5.08-1.25-.27-2.48-1-3.5.28-1.15.28-2.35 0-3.5 0 0-1 0-3 1.5-2.64-.5-5.36-.5-8 0C6 2 5 2 5 2c-.3 1.15-.3 2.35 0 3.5A5.403 5.403 0 0 0 4 9c0 3.5 3 5.5 6 5.5-.39.49-.68 1.05-.85 1.65-.17.6-.22 1.23-.15 1.85v4"
+            ></path>
+            <path d="M9 18c-4.51 2-5-2-7-2"></path>
+          </svg>
+          View on Github
+        </a>
+        <p class="mx-auto mt-4 text-gray-500 sm:w-2/3 lg:w-1/3">
+          If you find this library useful, please donate via
+          <a
+            href="https://www.paypal.me/alaind831"
+            target="_blank"
+            class="text-indigo-500 transition-colors hover:text-indigo-600"
+            >PayPal</a
+          >
+          or
+          <a
+            href="https://www.venmo.com/adumesny"
+            target="_blank"
+            class="text-indigo-500 transition-colors hover:text-indigo-600"
+            >Venmo (adumesny)</a
+          >
+          and help support it!
+        </p>
+      </div>
+    </section>
+    <!-- End hero section -->
+    <section class="pt-40 mx-auto max-w-7xl">
+      <p
+        class="inline px-4 py-1 font-medium tracking-tight border border-gray-700 rounded-full"
+      >
+        Workflow
+      </p>
+      <h2 class="mt-4 text-4xl font-medium">Framework Agnostic</h2>
+      <p class="mt-4 font-light">
+        The framework is designed to be used with any PHP framework. It is a set
```

Comment:
this text is wrong,a nd duplicated. nothing to do with PHP, and it's pure Typescript (not just JS).
I'll re-write it back. also never heard of Laravel so not listing it here... Angular/React/Vue are the big 3 framework today so I do want to spell those out.

Reply:
You're totally right, my mistake, I was focused on the style, thanks for the changes you've done